src/lib/evil/unposix/strings.h

```diff
@@ -3,10 +3,10 @@
 
 #include_next<string.h>
 
-#define strncasecmp _strnicmp
-#define strcasecmp _stricmp
-#define strncasecmp_l _strnicmp_l
-#define strcasecmp_l _strncmp_l
+// #define strncasecmp _strnicmp
+// #define strcasecmp _stricmp
+// #define strncasecmp_l _strnicmp_l
+// #define strcasecmp_l _strncmp_l
 
-inline int ffs(int i);
```

Comment:
What made those not important?

Reply:
already defined at `src/lib/evil/evil_strings.h`

Reply:
hmm, okay, so why not delete it?

Reply:
done: b733c67